website/screens/List.js

```diff
@@ -252,31 +267,48 @@ function getIconsForQuery(query, iconFamilyFilters) {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
+    minHeight: "100%",
+  },
+  inputContainer: {
+    flex: 1,
+    position: "relative",
+    // zIndex: 1000,
+    flexDirection: "row",
+    borderColor: "#2A2C33",
+    width: "100%",
   },
   searchContainer: {
+    flex: 40,
     flexDirection: "row",
     alignItems: "center",
     paddingHorizontal: 10,
     paddingVertical: 10,
     backgroundColor: "#2A2C33",
   },
   input: {
+    marginLeft: 20,
     padding: 10,
+    paddingLeft: 0,
     width: "100%",
     fontSize: 18,
     color: "#fff",
-    outlineColor: "#2A2C33",
+    outlineWidth: 0,
+  },
+  focussed: {
+    borderBottomColor: "#fff",
+    borderBottomWidth: 1,
```

Comment:
```suggestion
  focused: {
    borderBottomColor: "#fff",
```

Let's move the border width to generic input state and set color of border to transparent, so there is no component resize on interaction (+ typo fix `focused`).